### ISSUE TITLE
No way to clear Words

### ISSUE THREAD

**Issue by vadimkantorov** [skritter/skritter-html5#445]
https://skritter.com/words has no way to delete single words or all words without banning them.

**Comment by vadimkantorov** [comment]
Yep, for me it was the case of starting over after a three years break, so I indeed wanted to reset all learned words without banning them. At least, a pointer on the words page to "Remove my data" would be helpful. 

Otherwise, forgetting progress on selected words without banning them seems in itself useful.

**Comment by prokopcm** [comment]
That's a good feature suggestion for the text on the Words page. Thanks for pointing out the problems you had, we'll get some better signage in there so it'll be more obvious to future learners.
